### DISCUSSION TITLE
wholeBrainSeg_Large_UNEST_segmentation getting noise in the segmentation

### DISCUSSION THREAD

**Discussion by kpbhat25** [Project-MONAI/model-zoo#557] [Announcements]
**Describe the bug**
I am getting other parts from the brain MRI being segmented.Tried with a simple Oasis(MRI) data.
![Screenshot (32)](https://github.com/Project-MONAI/model-zoo/assets/89582705/9bbc5149-dd62-47f0-913f-3bb45d291712)

What things should I change to get a cleaner segmentation, as mentioned in the paper

**Comment by KumoLiu** [comment]
Hi, @kpbhat25, thanks for your interest here.

Several things you can try. The first and most reasonable thing is to improve the performance of the network. Also, you can try to add some post-processing such as keeping the largest connected component or adding some intensity filter if you find that the noise is at some similar intensity level.

Hope it helps, thanks.

**Comment by anuragp2018** [comment]
I faced the same issue. Is there any solution?

**Comment by abhishekpatil32** [comment]
I am also facing the same issue @KumoLiu. Was anyone able to solve this issue?
